hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/HazelcastTable.java

```diff
@@ -50,7 +50,7 @@
 import static java.util.stream.Collectors.joining;
 
 /**
- * Base class for all tables in the Calcite integration:
+ * Base class for all IMap tables in the Calcite integration:
```

Comment:
Why this change? It was correct and it is not anymore.

Reply:
I had wrong feeling about differences between `HazelcastTable` and `JetTable`, and tried to unify them. Then I realized  that they are different, but haven't change the comment. Thanks for catching.